x/nft/client/rest/tx.go

```diff
@@ -135,81 +110,3 @@ func editNFTMetadataHandler(cdc *codec.Codec, cliCtx context.CLIContext) http.Ha
 		clientrest.WriteGenerateStdTxResponse(w, cdc, cliCtx, baseReq, []sdk.Msg{msg})
 	}
 }
-
-type mintNFTReq struct {
-	BaseReq     rest.BaseReq `json:"base_req"`
-	Recipient   string       `json:"recipient"`
-	Denom       string       `json:"denom"`
-	TokenID     string       `json:"tokenID"`
-	Name        string       `json:"name"`
-	Description string       `json:"description"`
-	Image       string       `json:"image"`
-	TokenURI    string       `json:"tokenURI"`
-}
-
-func mintNFTHandler(cdc *codec.Codec, cliCtx context.CLIContext) http.HandlerFunc {
```

Comment:
@fedekunze Did we decide to leave the functions in here but just not allow them to be handled?

Reply:
in that case we would be having too much assumptions on the mint/burn nft logic, (eg mint only one NFT vs many, etc). Removing it allows devs to specify their own logic for them. Also it'd be dead code, so it's better if we add this to the NFT marketplace tutorial were it'd have a real use